e3/sys.py

```diff
@@ -235,18 +235,40 @@ def python_script(name, prefix=None):
     :return: a list that will be the prefix of your command line
     :rtype: list[str]
     """
+    def python_shebang(file_script):  # unix: no cover
+        """Return True if the script contains #!python shebang.
+
+        note: '#!/path/to/python' should return false, as we don't need to
```

Comment:
When producing relocatable python distribution we change the shebang to #!python. In that case prefix the command line with the current python interpreter.